build.sbt

```diff
@@ -31,7 +31,7 @@ libraryDependencies ++= Seq(
     exclude("org.bouncycastle", "bcprov-ext-jdk15on")
     exclude("org.bouncycastle", "bcutil-jdk15on")
     exclude("org.bouncycastle", "bcpkix-jdk15on"),
-  "org.broadinstitute.dsde.workbench" %% "workbench-notifications" % "0.3-25c2179"
+  "org.broadinstitute.dsde.workbench" %% "workbench-notifications" % "0.3-7189a695-SNAP"
```

Comment:
This will be updated to a proper version once https://github.com/broadinstitute/workbench-libs/pull/1206 merges